tests/test_file.py

```diff
@@ -178,6 +180,33 @@ def test_token_refresh_good_store(self):
     credentials._refresh(lambda x: x)
     self.assertEquals(credentials.access_token, 'bar')
 
+  def test_token_refresh_stream_body(self):
+    expiration = datetime.datetime.utcnow() + datetime.timedelta(minutes=15)
+    credentials = self.create_test_credentials(expiration=expiration)
+
+    s = file.Storage(FILENAME)
+    s.put(credentials)
+    credentials = s.get()
+    new_cred = copy.copy(credentials)
+    new_cred.access_token = 'bar'
+    s.put(new_cred)
+
+    valid_access_token = '1/3w'
+    token_response = {'access_token': valid_access_token, 'expires_in': 3600}
+    http = HttpMockSequence([
+        ({'status': '401'}, b'Initial token expired'),
```

Comment:
Good point; my comment was erroneous. Bad case of Tuesday afternoon brain. I now think these changes should have been to "str(httplib.symbolic_constant)" rather than to "httplib.symbolic_constant".

Does the test pass with ints instead of strings?